app/assets/javascripts/discourse/app/templates/modal/edit-slow-mode.hbs

```diff
@@ -15,18 +15,14 @@
 
   {{#if showCustomSelect}}
     <div class="control-group">
-      {{d-icon "hourglass-end"}}
-      {{input value=hours type="number" class="input-small" placeholder=(i18n "topic.slow_mode_update.hours")}}
-      {{input value=minutes type="number" class="input-small" placeholder=(i18n "topic.slow_mode_update.minutes")}}
-      {{input value=seconds type="number" class="input-small" placeholder=(i18n "topic.slow_mode_update.seconds")}}
-    </div>
-  {{/if}}
+      <label class="slow-mode-label">{{i18n "topic.slow_mode_update.hours"}}:</label>
```

Comment:
Small nitpick: The colon should be in the locale file instead of the template as it needs to be translatable as well.